src/System.Private.CoreLib/shared/System/Memory.cs

```diff
@@ -435,13 +435,13 @@ public unsafe MemoryHandle Pin()
                     // Array is already pre-pinned
```

Comment:
Aside: is there anyway to Debug.Assert that it is pinned?

Reply:
Nope